src/app/modules/perinatal/components/listado-perinatal.component.html

```diff
@@ -6,19 +6,36 @@
             <alertas-perinatal></alertas-perinatal>
         </plex-title>
         <plex-wrapper>
+
             <plex-datetime type="date" [(ngModel)]="fechaDesdeEntrada" name="fechaDesde" label="Fecha Desde"
                            class="fechas" [max]="fechaHastaEntrada" (change)="filtrar()"></plex-datetime>
             <plex-datetime type="date" [(ngModel)]="fechaHastaEntrada" name="fechaHasta" label="Fecha Hasta"
                            class="fechas" [min]="fechaDesdeEntrada" (change)="filtrar()"></plex-datetime>
+
             <plex-text [(ngModel)]="paciente" name="paciente" label="Buscar paciente" debounce="400"
                        (change)="filtrar()"></plex-text>
+
             <plex-datetime type="date" [(ngModel)]="fechaCita" name="fechaCita" label="Fecha de cita" class="fechas"
                            (change)="filtrar()">
             </plex-datetime>
+
             <plex-datetime type="date" [(ngModel)]="fechaUltimoControl" name="fechaControl" label="Último control"
                            class="fechas" (change)="filtrar()">
             </plex-datetime>
+
+            <div collapse>
+
+                <plex-select [(ngModel)]="profesional" name="profesional" label="Profesional"
+                             [data]="profesionales$ | async" (change)="filtrar()" labelField="nombre + apellido">
+                </plex-select>
+
+                <plex-select [(ngModel)]="organizacion" name="organizacion" label="Organizacion"
+                             [data]="organizaciones$ | async" (change)="filtrar()">
+                </plex-select>
```

Comment:
Hay un detalle en probando el filtro de profesional en DEMO, que varios registros tienen controles de N. Huenchuman pero esa profesional no aparece entrelas opciones del select. 
Para plex-select con datos de tablas maestras, recomiendo usar las directivas `tmProfesionales` y `tmOrganizaciones`. Tener en cuenta importar el `DirectiveLibModule` en el `perinatal.module.ts` y remover los Observables del component.
```suggestion
                <plex-select [(ngModel)]="profesional" name="profesional" label="Profesional"
                    tmProfesionales (change)="filtrar()" labelField="nombre + apellido">
                </plex-select>

                <plex-select [(ngModel)]="organizacion" name="organizacion" label="Organizacion"
                    tmOrganizaciones (change)="filtrar()">
                </plex-select>
```